src/dlgConnectionProfiles.cpp

```diff
@@ -1159,24 +1247,21 @@ bool dlgConnectionProfiles::hasCustomIcon(const QString& profileName) const
     return QFileInfo::exists(mudlet::getMudletPath(mudlet::profileDataItemPath, profileName, qsl("profileicon")));
 }
 
-void dlgConnectionProfiles::loadCustomProfile(const QString& profileName) const
+void dlgConnectionProfiles::loadCustomProfile(const QString& profileName, QMap<QString, QListWidgetItem*>& profilesMap) const
 {
     auto pItem = new QListWidgetItem();
+    pItem->setData(csmHiddenRole, false);
+    reduceFontSize(pItem);
     setItemName(pItem, profileName);
-
     setCustomIcon(profileName, pItem);
-    auto description = getDescription(profileName);
-    if (!description.isEmpty()) {
-        pItem->setToolTip(utils::richText(description));
-    }
-    profiles_tree_widget->addItem(pItem);
+    profilesMap.insert(profileName, pItem);
 }
 
-void dlgConnectionProfiles::setCustomIcon(const QString& profileName, QListWidgetItem* profile) const
+void dlgConnectionProfiles::setCustomIcon(const QString& profileName, QListWidgetItem* pItem) const
```

Comment:
I've renamed the `QListWidgetItem` argument here and elsewhere to the same name to make it easier to track when one is modified/used.